documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,59 @@
 # Install the Clear Containers runtime on RHEL 7.3
 
+## Requirements
 
-## Introduction
+In order to run cc-oci-runtime, you will need docker. Currently cc-oci-runtime supports docker 1.12.1 to 1.13.
```

Comment:
Please capitalize proper nouns such as Docker. Also, add the version number as Docker publishes it: v1.12.1 to v1.13.